src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudServiceTest.kt

```diff
@@ -200,6 +200,118 @@ class CosmasGoogleCloudServiceTest {
         assertNotNull(stream.error)
     }
 
+    @Test
+    fun simpleAddPatch() {
+        createVersion("file", "1", "1")
+        addPatchToService("abc", "-", "1", 1, "1")
+        val list = this.service.getPatchList("1", "1")
+        assertFalse(list == null)
+        assertEquals(1, list!!.size)
+        assertEquals("abc", list[0].text)
+        assertEquals("-", list[0].userId)
+        assertEquals(1L, list[0].timeStamp)
+        commit("1")
+        createVersion("fileNew", "1", "1")
+        addPatchToService("abcNew", "-New", "1", 10, "1")
+        val list1 = this.service.getPatchList("1", "1")
+        assertFalse(list1 == null)
+        assertEquals(1, list1!!.size)
+        assertEquals("abcNew", list1[0].text)
+        assertEquals("-New", list1[0].userId)
+        assertEquals(10L, list1[0].timeStamp)
+    }
+
+    @Test
+    fun checkEmptyList() {
```

Comment:
Cool test, thanks. Can you add similar check that version in memory becomes null wfter committing?

Reply:
add